Sample Data/Custom/CyberpionActionItems_CL.json

```diff
@@ -0,0 +1,234 @@
+{
+  "value": [
```

Comment:
sample data file should start with an object array, like below
**[
    {
      "TenantId": "93fc153b-9eb9-4c04-9e00-b9bbdcb4ae32",**

Please refer any existing sample data file

Reply:
True, I didn't notice - fixed it